docs/setup/install/install-devtron-helm-2.md

```diff
@@ -99,6 +99,8 @@ You will get result something like below
 
 The hostname mentioned here \( aaff16e9760594a92afa0140dbfd99f7-305259315.us-east-1.elb.amazonaws.com \) is the Loadbalancer URL where you can access the Devtron dashboard.
 
+Incase you don't see the results or service doesnt exist, it means devtron is still installing, please check again after 5 minutes.
```

Comment:
Incase you don't see the results or get a message "service doesn't exists", it ...